solutions/usaco-808

```diff
@@ -0,0 +1,70 @@
+---
+id: usaco-808
+source: USACO Bronze 2018 February
+title: Hoofball
+author: Ryan Chou
+---
+
+## Solution 1
+[Official Analysis](http://www.usaco.org/current/data/sol_hps_bronze_jan17.html)
+
+Find all source and oscillating cows and give a ball to each one.
```

Comment:
i don't think the official analysis mentions "oscillating cows" (?), so could you clarify that they are two cows that only pass to each other?